src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -363,7 +363,6 @@ well as menu structures (for main menu and popup menus).
          <menu label="_Import Dataset">
             <cmd refid="importDatasetFromFile"/>
             <cmd refid="importDatasetFromURL"/>
-            <cmd refid="importDatasetFromCSV"/>
```

Comment:
Yes, we do want this on the main menu. However, since this feature gets activated only with the `use_dataimport` feature flag, I'm removing it from the main menu (it will be still accessible from the environment pane if `use_dataimport` is set to `true`). There is additional work (looks non-trivial) to hide a menu across all our supported platforms based on a feature flag; therefore, seems reasonable to remove it now and bring it back later. If you think it's worth the work to figure out how to enable feature flags in the main menu, let me know and I'll spend some time here.